src/wp-admin/css/about.css

```
@@ -411,6 +412,13 @@
 	color: inherit;
 }
 
+.about__container .no-text-wrap h1,
+.about__container .no-text-wrap h2,
+.about__container .no-text-wrap h3,
+.about__container .no-text-wrap h4 {
+	text-wrap: unset;
+}
```

```suggestion
.about__container :is(h1, h2, h3, h4):lang(ja) {
	text-wrap: unset;
}
```

@AKSHAT2802 Sorry for the confusion. As [this comment](https://github.com/WordPress/wordpress-develop/pull/7209#issuecomment-2305287152) says, it seems best to solve the issue only with CSS selectors. Could you apply this suggestion and revert the changes to the PHP file?